### DISCUSSION TITLE
Sample Figma/Token Studio JSON?

### DISCUSSION THREAD

**Discussion by mikestecker** [terrazzoapp/terrazzo#170] [General]
Do you have a sample Figma or Token Studio JSON that I could use as a starting point at least for trying to work on a Chakra-UI plugin using? I have one that someone else created but I don't think they structured it properly and it's erroring left and right when trying to run the build command.

**Comment by drwpow** [comment]
Unfortunately, no, I haven’t seen any. Cobalt should be giving clear parser errors if anything matches the format, unless it’s just flat-out invalid JSON or YAML. But those should (hopefully) be easy-ish to fix? Happy to take a look / fix errors myself if you post the JSON you have.
